src/Chapter3/employee_table.cpp

```diff
@@ -5,30 +5,30 @@
 #include <vector>
 
 void EmployeeTable::addEmployee(const Employee &employee) {
-    employees.push_back(employee);
+    employees_.push_back(employee);
 }
 
-void EmployeeTable::removeEmployee(const std::string &employeeName) {
-    for (auto it = employees.begin(); it != employees.end(); ++it) {
-        if (it->getName() == employeeName) {
-            employees.erase(it);
+void EmployeeTable::removeEmployee(const std::string &name) {
+    for (auto it = employees_.begin(); it != employees_.end(); ++it) {
+        if (it->getName() == name) {
+            employees_.erase(it);
             break;
         }
     }
```

Comment:
```suggestion
std::erase_if(employees_, [](const Employee &e){e.getName() == name;});
```
C++20 ftw!

Reply:
I'm afraid this repo is stuck on C++14, at least for now. I'd *mayyyybe* consider bumping it to C++17 if we had a use for it, but I'm a bit concerned that C++20 might be a bit too bleeding edge for some of the compilers students will have installed :disappointed:.

It's surprising that `std::erase_if` took so long to make it into the standard though!

Reply:
Ah, I see. We could do the following, since C++11:

```cpp
emplloyees_.erase(
    std::remove_if(employees_.begin(), employees_.end(),
    [](const Employee &e){e.getName() == name;}), employees_.end();
```
Just doesn't look as nice :(

Reply:
Yeah. C++ often ends up looking rather ugly :disappointed: